app/fr/maif/izanami/web/SearchController.scala

```diff
@@ -110,11 +130,11 @@ class SearchController(
                 val parts   = parent.split("_")
                 val project = parts.head
 
-                // We look for shortes local context parent, since before him all contexts will be global
+                // We look for shorts local context parent, since before him all contexts will be global
```

Comment:
No, actually we look for "the shortest" local context parent, sorry for the typo?